app/models/consignment_inquiry.rb

```diff
@@ -0,0 +1,5 @@
+class ConsignmentInquiry < ApplicationRecord
```

Comment:
Is there a better name for this? "Inquiries" already have a meaning across the platform and I fear this will lead to confusion.

Reply:
I actually named it ConsignmentInquiry as a way to distinguish this inquiry from the inquiries we have across platform. I can't think of a better name. 
Any suggestions? 🙏

Reply:
I don't have a better suggestion. As long as you tried I guess. 😞